packages/mina-service/src/domain/rollup-offchain.ts

```diff
@@ -70,36 +70,49 @@ export class RollupOffChain {
       );
     }
 
-    // If previous proof not found and operationNumber must be 1, which mean first time create
-    if (!previousProof && task.operationNumber === 1) {
+    if (!previousProof) {
+      // If previous proof not found and operationNumber must be 1, which mean first time create
```

Comment:
"Initial off-chain rollup" is better in clarification compare to "first time create".